scipy/optimize/__init__.py

```diff
@@ -255,12 +255,14 @@
 
    linprog_verbose_callback -- Sample callback function for linprog (simplex).
 
-Assignment problems:
+Assignment problems
+===================
 
 .. autosummary::
    :toctree: generated/
 
    linear_sum_assignment -- Solves the linear-sum assignment problem.
+   quadratic_assignment -- Solves the quadratic assignment problem.
```

Comment:
I'm not sure what the convention is here, but I think I would write

"Approximately solves the quadratic assignment problem"

And only say solves if we have an algorithm that finds the global optimal exactly with certainty?